app/static/lib/conp-react/src/CbrainModalPipeline/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from "react";
+import * as ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+const CbrainModalPipeline = (props) => {
+  const { platforms, title, cbrainIds } = props;
+  const finish = (event) => {
+    $("#cbrainModal").modal("hide");
+  };
+  const baseUrl = platforms[0].uri;
+  const getDatasetId = (datasetUrl) =>
+    `switch_group_id=${datasetUrl.split("?id=")[1]}`;
+  useEffect(() => {
+    $("#cbrainModal").modal("handleUpdate");
+    return;
+  });
+  return ReactDOM.createPortal(
+    <div>
+      <p>
+        You're about to load the pipeline <strong>{title}</strong> on
+        CBRAIN. Please select a dataset to process with this pipeline (or "None"
+        to load only the pipeline).
+      </p>
+      <p>Note that not all pipelines are compatible with all datasets.</p>
```

Comment:
This sentence is only present for the pipeline modal, not in the dataset modal. I guess it should be represented on both?